medium-editor.js

```diff
@@ -44,4 +37,13 @@ export default {
     this.$root.mediumEditor.removeElements(this.$refs.element)
   }
 
+  methods: {
+
+	// impure helper function, replaces some element from the dom with a new one of the given tag name
+	// returns the new one.
+	function replaceElementWith (element, tagName) {
```

Comment:
First off, thank you very much for your help :)

Secondly: is the function keyword really required? As you can see, the other member function keywords are not written out. Let's keep that consistent :)

If you want, you may add yourself to some contributor list in the readme.